exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/join/TestHashJoinAdvanced.java

```diff
@@ -197,4 +199,14 @@ public void emptyPartTest() throws Exception {
       BaseTestQuery.resetSessionOption(ExecConstants.SLICE_TARGET);
     }
   }
+
+  @Test // DRILL-6089
+  public void testJoinOrdering() throws Exception {
+    final String query = "select * from dfs.`sample-data/nation.parquet` nation left outer join " +
```

Comment:
I missed this in the prior review but I think the ORDER BY should be on the column coming from the left input of the join (assuming that nation table is the left input).  The reason is the original code was using 'convertedLeft' to propagate the collation trait from the Probe (left) side, so the test should match that scenario. Another thing is the you don't necessarily need the subquery.. you could  do  `select * from  nation.parquet nation  left outer join region.parquet region on ... order by nation.n_name desc`.

Reply:
Thanks for catching this, updated the test.